docs/dev-tools/grokdebugger/index.asciidoc

```diff
@@ -1,25 +1,114 @@
 [role="xpack"]
 [[xpack-grokdebugger]]
-== Debugging Grok Expressions
+== Debugging grok expressions
```

Comment:
I wonder if the current title is the best fit. Since Console is just "Console", should this title be "Grok debugger" to match the name of the tool in the UI?

Reply:
Good suggestion to make the titles parallel.  I think a good approach would be to change the Console title to make it more descriptive.